rotkehlchen/tasks/manager.py

```diff
@@ -141,6 +141,8 @@ def _maybe_schedule_db_upload(self) -> Optional[gevent.Greenlet]:
         assert self.premium_sync_manager is not None, 'caller should make sure premium sync manager exists'  # noqa: E501
         if self.premium_sync_manager.check_if_should_sync(force_upload=False) is False:
             return None
+
+        log.debug('Scheduling task for DB upload to server')
```

Comment:
Why are these both logs needed? If I understand correctly, they will always be printed one after another

Reply:
Which both logs?

Reply:
This and that one

![image](https://user-images.githubusercontent.com/63492346/198568802-a79f9d06-e04c-4940-803a-386f052bce82.png)

Reply:
No they are **very very** different and we need both.

And not having them was what made it hard to debug.

One log is in the scheduling greenlet.
Other log is in the task, and will appear whenever the task is scheduled to start.

And absolutely not, they are not going to print one after the other. There is no guarantee for that. In fact the order in which they print is important and helped me debug this.

Reply:
Alright :+1: